packages/charting/src/components/Legends/Legends.base.tsx

```diff
@@ -238,12 +241,21 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
         onBlur={onMouseOut}
         data-is-focusable={true}
       >
-        <div className={classNames.rect} />
+        <div className={this._getShapeClass(classNames, legend)} />
         <div className={classNames.text}>{legend.title}</div>
       </div>
     );
   };
 
+  private _getShapeClass(classNames: IProcessedStyleSet<ILegendsStyles>, legend: ILegend): string {
+    switch (legend.shape) {
```

Comment:
can probably shorten with an if/else since there's only one case

Reply:
change it to if/else.